### PR TITLE
Bump minimum versions of optional dependencies during installation

### PR DESCRIPTION
**Pull request by jguittard**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | yes
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

While trying to install skeleton on PHP 8.3.x environment, current `laminas-log` dependency (2.16) fails. This PR relies on `laminas/laminas-log:2.17` and bumps latest stable dependencies on `laminas-skeleton-installer`

### CODE REVIEW

File: .gitattributes
Comment (alexmerlin) [inline .gitattributes:1]: All files should end with an empty line - you should revert this file.

**Review by alexmerlin — CHANGES_REQUESTED** [review]
Other than the tiny issue with the line ending in `.gitattributes`, LGTM!

File: .gitattributes
Comment (jguittard) [reply .gitattributes:1]: Done

**Comment by samsonasik** [comment]
You can read how to pass DCO https://github.com/laminas/laminas-mvc-skeleton/pull/87/checks?check_run_id=31607581934

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ data/cache/*
 !data/cache/.gitkeep
 phpunit.xml
 composer.lock
+.idea/
```
Comment (gsteel) [inline .gitignore]: This belongs in your global `~/.gitignore_global` file ;)

**Review by gsteel — CHANGES_REQUESTED** [review]

**Review by gsteel — APPROVED** [review]
Thank you @jguittard :)

**Pull request merged**